agent/util-scripts/pbench-tool-meister-start

```diff
@@ -0,0 +1,542 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+"""pbench-tool-meister-start
+
+Responsible for:
+
+   1. Starting a redis server
+   2. Loading tool group data for the given tool group into the redis server
+   3. Starting the tool-data-sink process
+   4. Starting all the local and remote tool meisters
+
+When complete we leave running, locally, a redis server and a tool data sink
+process, and any local or remote tool meisters.
+
+The pbench-tool-meister-stop script will take care of (gracefully) stopping
+all of these processes, locally or remotely.
+"""
+
+import sys
+import json
+import os
+import signal
+import time
+import errno
+import redis
+import logging
+from distutils.spawn import find_executable
+from pathlib import Path
+
+
+# Port number is "One Tool" in hex 0x17001
+# FIXME: move to common area
+redis_port = 17001
```

Comment:
This obviously needs to be encapsulated somewhere; a hypothetical PbenchRedis wrapper class wouldn't be a bad place. ;-)

Reply:
Don't worry about it now; something for the list if we do centralize/encapsulate the Redis idioms, though.

Reply:
Tracked via issue #1781.

Reply:
Tracked via issue #1781.